src/zzz_od/api/utils/validation.py

```
@@ -0,0 +1,242 @@
+            raise ValidationError(f"不支持的手柄类型: {gamepad_type}，支持的类型: {', '.join(supported_types)}")
+
+
+class FileOperationHelper:
+    """文件操作助手"""
+
+    @staticmethod
+    def ensure_directory_exists(directory_path: str) -> None:
+        """确保目录存在"""
+        try:
+            Path(directory_path).mkdir(parents=True, exist_ok=True)
+        except PermissionError as e:
```

Ruff (0.12.2)

Local variable `e` is assigned to but never used

Remove assignment to unused variable `e`

(F841)